src/contract_wrappers/modules/transfer_manager/manual_approval_transfer_manager_wrapper.ts

```diff
@@ -253,8 +267,21 @@ export default class ManualApprovalTransferManagerWrapper extends ModuleWrapper
   };
 
   public addManualApprovalMulti = async (params: AddManualApprovalMultiParams) => {
+    assert.assert(await this.isCallerAllowed(params.txData, Perms.TransferApproval), 'Caller is not allowed');
     assert.isETHAddressHexArray('from', params.from);
     assert.isETHAddressHexArray('to', params.to);
+    assert.checkAddManualApprovalMultiConditions(
+      params.from,
+      params.to,
+      params.allowances,
+      params.expiryTimes,
+      params.descriptions,
+    );
+    /* eslint-disable no-await-in-loop */
```

Comment:
We should not disable eslint rule.
We should take advantage of Promise.all instead. Check https://eslint.org/docs/rules/no-await-in-loop